eng/generators.targets

```diff
@@ -1,68 +1,56 @@
-      OutputItemType="Analyzer"
-      ReferenceOutputAssembly="false" />
-  </ItemGroup>
-  <ItemGroup Condition="'@(EnabledGenerators)' != ''
-                        and @(EnabledGenerators->AnyHaveMetadataValue('Identity', 'LibraryImportGenerator'))
-                        and '$(IncludeLibraryImportGeneratorSources)' == 'true'">
-
+  <ItemGroup Condition="'@(EnabledGenerators)' != '' and
+                        @(EnabledGenerators->AnyHaveMetadataValue('Identity', 'LibraryImportGenerator'))">
     <!-- Only add the following files if we are not on the latest TFM. -->
-    <Compile Condition="'$(NetCoreAppCurrentTargetFrameworkMoniker)' != '$(TargetFrameworkMoniker)'"
-             Include="$(CoreLibSharedDir)System\Runtime\InteropServices\LibraryImportAttribute.cs" />
-    <Compile Condition="'$(NetCoreAppCurrentTargetFrameworkMoniker)' != '$(TargetFrameworkMoniker)'"
-             Include="$(CoreLibSharedDir)System\Runtime\InteropServices\StringMarshalling.cs" />
+    <Compile Include="$(CoreLibSharedDir)System\Runtime\InteropServices\LibraryImportAttribute.cs;
+                      $(CoreLibSharedDir)System\Runtime\InteropServices\StringMarshalling.cs"
+             Condition="'$(TargetFrameworkMoniker)' != '$(NetCoreAppCurrentTargetFrameworkMoniker)'" />
   </ItemGroup>
 
-  <ItemGroup>
-    <EnabledGenerators Include="RegexGenerator" Condition="'$(EnableRegexGenerator)' == 'true'" />
+  <!-- Use this complex item list based filtering to add the ProjectReference to make sure dotnet/runtime stays compatible with NuGet Static Graph Restore. -->
+  <ItemGroup Condition="'@(EnabledGenerators)' != '' and
+                        @(EnabledGenerators->AnyHaveMetadataValue('Identity', 'LibraryImportGenerator'))">
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.InteropServices\gen\LibraryImportGenerator\LibraryImportGenerator.csproj;
+                                $(LibrariesProjectRoot)System.Runtime.InteropServices\gen\Microsoft.Interop.SourceGeneration\Microsoft.Interop.SourceGeneration.csproj"
+                      OutputItemType="Analyzer"
+                      ReferenceOutputAssembly="false" />
   </ItemGroup>
 
-  <!-- Use this complex ItemGroup-based filtering to add the ProjectReference to make sure dotnet/runtime stays compatible with NuGet Static Graph Restore. -->
-  <ItemGroup Condition="'@(EnabledGenerators)' != ''
-                        and @(EnabledGenerators->AnyHaveMetadataValue('Identity', 'RegexGenerator'))">
-    <ProjectReference
-      Include="$(LibrariesProjectRoot)System.Text.RegularExpressions/gen/System.Text.RegularExpressions.Generator.csproj"
-      OutputItemType="Analyzer"
-      ReferenceOutputAssembly="false" />
+  <ItemGroup Condition="'$(EnableRegexGenerator)' == 'true'">
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Text.RegularExpressions\gen\System.Text.RegularExpressions.Generator.csproj"
```

Comment:
Don't we always try to use Unix-style folder separators everywhere?

Reply:
I don't think so:
- https://github.com/dotnet/runtime/blob/0533792282ba5546e10cb5c3d10748aff46ddc1a/src/libraries/System.Console/ref/System.Console.csproj#L9
- https://github.com/dotnet/runtime/blob/0533792282ba5546e10cb5c3d10748aff46ddc1a/src/libraries/Microsoft.Extensions.Http/src/Microsoft.Extensions.Http.csproj#L29-L32
- https://github.com/dotnet/runtime/blob/0533792282ba5546e10cb5c3d10748aff46ddc1a/src/libraries/System.IO.Ports/src/System.IO.Ports.csproj#L23-L37